src/sieppari/async/ext_lib_support.cljc

```diff
@@ -0,0 +1,27 @@
+(ns sieppari.async.ext-lib-support
+  #?(:cljs (:require-macros [sieppari.async.ext-lib-support])))
+
+(defmacro available?
```

Comment:
This macro does not work in `lumo`, will try in vanilla ClojureScript

Reply:
The macro is not needed in Cljs because conditional requires can't be used there anyway.

Reply:
Yep I will avoid requiring the whole namespace for now. We should probably have folks explicitly require a namespace that trigger `extend-protocol` on the right stuff (core.async channels) for ClojureScript instead. What do you think the best solution is?